gtfs/spec/en/reference.md

```diff
@@ -407,14 +407,14 @@ File: **Optional**
 
 Primary Key (`fare_product_id`, `fare_media_id`)
 
-To describe the different types of tickets or fares that can be purchased by riders.
+To describe the different types of tickets or fares, including those directly purchasable by riders and integral virtual entities like transfer costs.
 
 |  Field Name | Type | Presence | Description |
 |  ------ | ------ | ------ | ------ |
-| `fare_product_id` | ID | **Required** | Identifies a fare product. |
+| `fare_product_id` | ID | **Required** | Identifies a fare product.<br><br>Multiple records in [fare_products.txt](#fare_productstxt) may have the same `fare_product_id`, in which case all fare products with that ID will be applied when referenced from another file. |
```

Comment:
Is it worth being more explicit about the interaction with `fare_media_id` here?  Which is to say, multiple records maybe the same `fare_product_id` with different fare media, indicating different methods for purchasing and validating the same fare product.

Reply:
Yes, thanks for this suggestion @bdferris-v2!

Change made - https://github.com/google/transit/pull/426/commits/f0a12b8deb528ee804e2120c4b9be1fb2ddfa07b

Does it look clearer now?